angular-workspace/projects/ni/nimble-angular/src/directives/rich-text-viewer/nimble-rich-text-viewer.directive.ts

```diff
@@ -0,0 +1,25 @@
+import { Directive, ElementRef, Input, Renderer2 } from '@angular/core';
```

Comment:
We are in the very early stages of reorganizing nimble-angular to expose a separate secondary entry point for each component. This work is tracked by #172 and was started for just the table component in #1252. I think we should attempt to follow the new layout for this new component rather than having to move it later (a breaking change) when we complete work on that issue.

Could you try following the layout from that PR's description? If we run into major issues we could debate doing this now but I think it'll save time in the long run to start doing it from the beginning.

Reply:
Thanks for letting us know. We are unaware of this new layout to expose a secondary entry point for each component. I changed it into a new layout and verified the working of the directive in the example-client-app. Please review the changes and let me know your comments.

Reply:
The new entry point looks good to me. Thanks for making that change.